.github/workflows/ci-e2e-checks.yaml

```diff
@@ -109,12 +109,13 @@ jobs:
       run: |
         make ns-create-test
 
-    - name: Install Tools 
+    - name: Install Tools
       timeout-minutes: 5
       run: |
         sudo add-apt-repository ppa:rmescandon/yq
         sudo apt update
         sudo apt install yq -y
+        yq --version
```

Comment:
**question:** Is this meant to remain for troubleshooting CI issues in the future or is this a debug line which was meant to be removed?

Reply:
Yes, I intend to leave this in going forward as the version isn't pinned and can change whenever the underlying action image changes. It's helpful to be able to matchup the local version without having to add this , run CI, then remove it later.